masschallenge/django-accelerator
# [AC-5600] - Using Full Bio in New Directory

**Changes Introduced in [AC-5600](https://masschallenge.atlassian.net/browse/AC-5600):**

- Add a Validator on field ExpertProfile.bio to be no longer than 7500 characters.

**Background:**
- We discovered that the reason the bio did not index in the first place was because we had a few outliers with overly long bio. There are currently 28 users with bios longer then 7500 characters (the limit we've reached is somewhere above 16000), and @EcoLek decided that this new limit is more than enough for a bio. @EcoLek will follow up with violating users re trimming down their current bio (list of users is on the ticket).

**Test:**
- On a production dump
- Go to http://localhost:8181/admin/simpleuser/user/14430/change/ (one of the violating users)
- Try to make a change in the profile data, and click save.
- You will see a validation error on the bio field.
- Trim the bio down to below 7500 characters, and click save. You will succeed.
